_pkgdown.yml

```diff
@@ -9,6 +9,9 @@ template:
       <script defer data-domain="duckplyr.tidyverse.org,all.tidyverse.org" src="https://plausible.io/js/plausible.js"></script>
 
 reference:
+- title: Methods for dplyr generics
```

Comment:
Perhaps put this underneath "Data ingestion"?

Reply:
Wouldn't it make the methods harder to find and notice?